src/libsyntax_ext/deriving/debug.rs

```diff
@@ -40,7 +40,7 @@ pub fn expand_deriving_debug(cx: &mut ExtCtxt,
                           name: "fmt",
                           generics: LifetimeBounds::empty(),
                           explicit_self: borrowed_explicit_self(),
-                          args: vec![fmtr],
+                          args: vec![(fmtr, "_f")],
```

Comment:
probs should be `fmt`

(check what the trait uses)

Reply:
Oh, I know why it's short; it's to encourage implementors to keep it short. Leave it `f` then.